### PR TITLE
fix(components/headerbar): removed comma from classname

### PR DESCRIPTION
**Pull request by sdwvit**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
classname generated with comma, some browsers do not strip commas

**What is the new behavior?**
ff is happy with this no-comma classname

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://392.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://392.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://392.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://392.talend.surge.sh)

File: packages/components/src/HeaderBar/HeaderBar.component.js [hunk 1]
```diff
@@ -153,7 +153,7 @@ Products.propTypes = {
 
 function HeaderBar(props) {
 	return (
-		<nav className={classNames([`tc-header-bar, ${theme['tc-header-bar']}`])} >
+		<nav className={classNames([`tc-header-bar ${theme['tc-header-bar']}`])} >
```
Comment (frassinier) [inline packages/components/src/HeaderBar/HeaderBar.component.js]: It produces undefined within tests, what do you think about 
```jsx
<nav className={classNames(theme['tc-header-bar'], 'tc-header-bar')}>
```

Comment (sdwvit) [inline packages/components/src/HeaderBar/HeaderBar.component.js]: +

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://392.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://392.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://392.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://392.talend.surge.sh)

**Pull request merged**